<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addRab</class>
 <widget class="QWidget" name="addRab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="1">
    <widget class="QDateEdit" name="dateEdit">
     <property name="currentSection">
      <enum>QDateTimeEdit::YearSection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy.MM.dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="QComboBox" name="comboBox_3"/>
   </item>
   <item row="13" column="2">
    <widget class="QDateEdit" name="dateEdit_3">
     <property name="displayFormat">
      <string>yyyy.MM.dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QLabel" name="label_10">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Дата приказа</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QComboBox" name="comboBox_6"/>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Должность работника</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Добавить работника</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QComboBox" name="comboBox_2"/>
   </item>
   <item row="15" column="1">
    <widget class="QLabel" name="label_7">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Помещение</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Добавление счетчика</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QDateEdit" name="dateEdit_2">
     <property name="displayFormat">
      <string>yyyy.MM.dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Срок службы</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Марка</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_8">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Название приказа</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QComboBox" name="comboBox_4"/>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Название отдела</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Имя работника</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Имя рабочего</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <widget class="QPushButton" name="pushButton_3">
     <property name="text">
      <string>Добавление приказа</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="11" column="0">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="10" column="2">
    <widget class="QLabel" name="label_9">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Связанный приказ</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="QComboBox" name="comboBox_5"/>
   </item>
   <item row="18" column="3">
    <widget class="QPushButton" name="pushButton_4">
     <property name="text">
      <string>Добавить отчет</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_11">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Серийный номер</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QComboBox" name="comboBox_7"/>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="label_12">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Дата отчета</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QDateEdit" name="dateEdit_4">
     <property name="displayFormat">
      <string>yyyy.MM.dd</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QLabel" name="label_13">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Название приказа</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="3">
    <widget class="QComboBox" name="comboBox_8"/>
   </item>
   <item row="14" column="3">
    <widget class="QLabel" name="label_14">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Работоспособность счетчика</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="15" column="3">
    <widget class="QComboBox" name="comboBox_9"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
